### DISCUSSION TITLE
Execution request

### DISCUSSION THREAD

**Discussion by sandeepellur1219** [cbeust/testng#2939] [General]
How can i run this project can you please help to provide a steps i  am using eclispe iDE

**Comment by krmahadevan** [comment]
@sandeepellur1219 - TestNG uses Gradle as the build tool. After you clone the repo, you can run:

* `./gradlew clean test` - To run all the tests
* `./gradlew autostyleApply` - To automatically format the code
* `./gradlew tasks` - To list all the gradle tasks that can be run.

Beyond this, there's nothing additional that is required for you to work with this codebase

**Comment by juherr** [comment]
We are using IntelliJ and the project is loaded like any other gradle project.

What is your issue?
